spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -96,6 +96,19 @@
     end
   end
 
+  describe "#unpublish" do
```

Comment:
You might need to rebase this against master, since I've also created tests for unpublishing in https://github.com/alphagov/collections-publisher/pull/774 😁